doc/common/chapter00.md

```diff
@@ -208,49 +210,49 @@ The CNTT work is not done in a vacuum.  The intention from the beginning was to
-- CNTT includes a traceability matrix in its **Reference Conformance** document detailing every test case (or group of test cases) available in the community and map them to the high level test case definition and the requirements they are fulfilling.
-- CNTT includes a testing Playbook in its **Reference Conformance** document detailing instructions of how to run **OPNFV** testing framework and test cases against commercial NFV products (infrastructure and workload) to check conformance to CNTT specifications. CNTT testing Playbook will also details instructions of how to submit testing results for the **OVP** badging process.
+- Anuket specifies lab requirements in the **Reference Implementation** document which will be used to define what labs can be used within the community for the purpose of installing and testing Anuket conformant cloud infrastructure implementations.
+- Anuket includes a lab Playbook in its **Reference Implementation** detailing available suitable labs to run and test cloud infrastructure implementations with access details and processes.
+- Anuket specifies installer requirements in the **Reference Implementation** document which will be used to determine a suitable installer to be used to install a **Reference Implementation** of cloud infrastructure that are conformant to Anuket specifications.
+- Anuket includes an installation Playbook in its **Reference Implementation** detailing instructions of how to install the reference implementation using community installers. The **Reference Implementation** created is expected to be fully conformant to Anuket specifications and must pass all test suites defined in the Anuket **Reference Conformance** document.
+- Anuket specifies testing framework requirements in the **Reference Conformance** document which will be used to determine a suitable testing framework and portals to be used for the purpose of running test suites and tools, and carry out badging processes.
+- Anuket defines high level test cases in the **Reference Conformance** document that relates to requirements coming from both the **Reference Model** and **Reference Architecture** to be used to determine what testing projects within the community are suitable to deliver those tests and convert them into low level test cases that can be implemented within those testing projects.
+- Anuket includes a traceability matrix in its **Reference Conformance** document detailing every test case (or group of test cases) available in the community and map them to the high level test case definition and the requirements they are fulfilling.
```

Comment:
No longer true as traceability matrix is now in RA documents. Suggest remove

Reply:
Please open another PR for removing outdated content.

Reply:
@EditKoselak I think "Reference Conformance" needs to be changed to "Reference Architecture"